.github/workflows/self-hosted.yml

```diff
@@ -49,7 +52,7 @@ jobs:
         id: nodemodulescache
         with:
           path: node_modules
-          key: ${{ runner.os }}-self-hosted-node-modules-${{ hashFiles('yarn.lock') }}
+          key: ${{ runner.os }}-self-hosted-node-modules-${{ hashFiles('yarn.lock', 'api-docs/yarn.lock', '.volta.json') }}
```

Comment:
I'm not sure all of the installs are the same -- wouldn't want the cache to flipflop based on the apidocs usage -- double check this (it might be fine already)

Reply:
good catch i meant to remove this